examples/step-31/step-31.cc

```diff
@@ -502,13 +513,13 @@ namespace Step31
     DoFHandler<dim>                     stokes_dof_handler;
     ConstraintMatrix                    stokes_constraints;
 
-    std::vector<types::global_dof_index> stokes_block_sizes;
+    std::vector<IndexSet>               stokes_block_sizes;
```

Comment:
I think the name of the variable is now no longer descriptive. Can you choose a better name?